ObsFacilityWikidata.tex

```diff
@@ -458,25 +470,27 @@ \section{More details on Wikidata}
 \begin{itemize}
 \item added relevant \emph{is instance of} statements, to ease the 
 record discovery and limit the complexity of the queries (see, e.g., 
-Appendix \ref{appendix:sparql}. 
+Appendix \ref{appendix:sparql}). 
 \item include \emph{external identifiers} statements, when values 
 are missing
 \item define new identifier authorities in Wikidata for lists that 
 are not yet registered to be used as \emph{external identifiers}. 
 \end{itemize}
-The first two items are easy to setup and solve when the record is 
-found. However, is the \emph{is instance of} statement is not 
-properly labeled in th first place, finding the record is not always 
-straightforward as commons names can be used in many other contexts. 
+The first two items are easy to set up and solve when the record is 
+found. However, if the object of the \emph{is instance of} statement is 
+wrong in the first place, finding the record is not always 
```

Comment:
I would not write _wrong_ here, since the logics might be different, depending on the record contributors.

Reply:
Fixed in 6393956